sysutils/ddrescueview/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+
+name            ddrescueview
+version         0.4.5
+revision        0
+categories      sysutils aqua
+license         GPL-2
+maintainers     {@kamischi web.de:karl-michael.schindler} openmaintainer
+description     Graphical viewer for GNU ddrescue log files.
+
+long_description \
+                This small tool allows the user to graphically examine ddrescue's log \
+                files in a user friendly GUI application. The Main window displays a \
+                block grid with each block's color representing the block types it \
+                contains. Many people know this type of view from defragmentation \
+                programs. The program is written in Object Pascal using the Lazarus IDE.
+
+homepage        https://sourceforge.net/projects/${name}
+master_sites    sourceforge:${name}/Test%20builds/v${version}/
+distname        ${name}-source-${version}
+use_xz          yes
+
+checksums       rmd160  aca84a816122891b4c824f02a27b2cf2da64908b \
+                sha256  57383c394e62612ce2a799438b00c6e3c465c31f9ba940e077f624e2e7028465 \
+                size    211184
+
+depends_build   port:lazarus port:makeicns
+
+patch {
+# remove incorrect linux linker options
+    reinplace "s|-z relro --as-needed||g" source/ddrescueview.lpi
+}
+
+use_configure   no
+
+build {
```

Comment:
Although I can see how this would work, this is not how we'd usually do this. Let me see if I can tweak it into the standard MacPorts' pattern of doing things...

Reply:
I was happy to have something working 😉

You mean something like this?

build.cmd lazbuild
build.target ddrescueview.lpi

Not sure about the part with makeicns. Maybe post-build?

Also, patching the app into post-build?

Reply:
yes, all that!